_data/sites.yml

```diff
@@ -1117,7 +1117,12 @@
   url: https://fedi.dev/
   size: 237
   last_checked: 2022-05-15
-
+  
+- domain: fem.lol
+  url: https://fem.lol/
+  size: 302
+  last_checked: 2023-9-22
```

Comment:
The month should have a leading 0. Do you mind fixing that?

Reply:
Oops, sorry about that. Its fixed